ci/Jenkinsfile.groovy

```diff
@@ -309,7 +313,7 @@ ansiColor('xterm') {
                                     unstash 'x86_64-manylinux-wheels'
                                     test_in_docker("x86_64-manylinux-py37", "37",
                                                    DOCKER_IMAGE_X86_64_MANYLINUX,
-                                                   needsLargerTest)
+                                                   needsLargerTests)
```

Comment:
I somehow think we don't need to pass this variable as a parameter -- it's global after all. Also, we don't pass any other variables like `doPpcTests` or `doExtraTests`.

Reply:
Probably yes. We can also do a lot of other re-factoring and fixes in this groovy file.

Reply:
feel free

Reply:
I've already fixed the `isModified()` routine, it actually never returned `true` as only checked the base file names and not the full file names to match the pattern. That is probably a reason why the larger tests were never triggered automatically and some broken tests leaked.